results/v0.42.0/m5.large-website.js-10s.txt

```diff
@@ -0,0 +1,1903 @@
+
```

Comment:
shouldn't the name be `10m` instead of `10s`

Reply:
That suffix indicates the `sleep()` period per iteration, which is a more distinguishing factor for some of these tests. Notice that we also refer to it in the graph titles, and the `README` sections. The actual test duration is not that important.